packages/client/components/ReflectionGroup/DraggableReflectionCard.tsx

```diff
@@ -112,28 +114,17 @@ const DraggableReflectionCard = (props: Props) => {
   const showDragCursor = isDraggable && canHandleDrag && !isEditing && !isDropping
   // slow state updates can mean we miss an onMouseDown event
   const handleDrag = canHandleDrag ? onMouseDown : undefined
-  // if spotlight was just opened and card is in the middle of dropping we let it drop into original position
-  const [isFinishingRemoteDragging, setIsFinishingRemoteDragging] = useState(
-    () => isDropping && isSpotlightOpen && !!remoteDrag
-  )
-  // if the card was finishing remote drag and it was dropped into the original position, let it behave normally
-  useEffect(() => {
-    if (isFinishingRemoteDragging && !isDropping) {
-      setIsFinishingRemoteDragging(false)
-    }
-  }, [isFinishingRemoteDragging, isDropping])
 
   return (
     <DragWrapper
       ref={(c) => {
-        if (isFinishingRemoteDragging) {
-          return
-        }
-        // if the spotlight is closed, this card is the single source of truth
+        // If the spotlight is closed, this card is the single source of truth
         // Else, if it's a remote drag that is not in the spotlight
         // Else, if this is the instance in the source or search results
+        // And Spotlight modal isn't entering. This throws off dropping remote card position
         const isPriorityCard =
-          !isSpotlightOpen || (!isReflectionIdInSpotlight && remoteDrag) || isInSpotlight
+          (!isSpotlightOpen || (!isReflectionIdInSpotlight && remoteDrag) || isInSpotlight) &&
+          !isSpotlightEntering
```

Comment:
I don't love passing an extra prop down through a few components, but `isSpotlightEntering` is descriptive, it means we can remove the `useEffect` logic above, and it fixes the two tricky bugs mentioned in the PR description